components/list/src/ListItem.js

```diff
@@ -0,0 +1,157 @@
+/* @jsx jsx */
+
+import React, { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, css, useTheme } from '@westpac/core';
+import { List } from './List';
+
+// ==============================
+// Utils
+// ==============================
+const getListIcon = ({ appearance, color, icon: Icon }) => {
+	const { list } = useTheme();
+
+	const style = {
+		position: 'absolute',
+		left: 0,
+		top: 2,
+	};
+
+	switch (appearance) {
+		case 'bullet':
+			return (
+				<svg
+					aria-labelledby="title-bullet-list"
```

Comment:
remove the `aria-labelledby` attribute since we aren't providing a `<title>` element